internal/validation/validate.go

```diff
@@ -212,7 +204,7 @@ func (v *APIRuleValidator) validateRules(ctx context.Context, client client.Clie
 			if err != nil {
 				problems = append(problems, Failure{
 					AttributePath: attributePathWithRuleIndex + ".service",
-					Message:       "No label selectors found for service",
+					Message:       "Couldn't get label selectors for service",
```

Comment:
should this stay? this is general validation on path level and not only for JWT case, you removed it in `validateService()` on line 174?

Reply:
I would say that this can stay, as we expect that a service is present anyway

Reply:
I guess we may want to skip failing the API Rule generation maybe? WDYT

Reply:
Would this work at runtime, if not, I would say we fail as early as possible.

Reply:
This would work, as we can create e.g VirtualService, just the service won't be present (that was how it worked before)
It only matters for Istio JWT whether the Service is actually there for the APs creation

Reply:
in this case I would be okay with just warning, as long as scenario works

Reply:
https://github.com/kyma-project/api-gateway/pull/402/commits/080bcc10892851990cc32a139cd164bbbf9f0731